app/charts/shared/axis-width-time.tsx

```diff
@@ -106,7 +113,11 @@ export const AxisTime = () => {
     <>
       <foreignObject
         x={margins.left + chartWidth / 2 - labelWidth / 2}
-        y={margins.top + chartHeight + 40}
+        y={
+          margins.bottom +
+          chartHeight +
+          (interactiveFiltersConfig?.timeRange.active ? 0 : 40)
+        }
```

Comment:
@noahonyejese I am not sure if the axis should know about interactive elements.

It looks like we already add the padding in `getChartPadding` function for the x axis label:

https://github.com/visualize-admin/visualization-tool/blob/71406b863161f2cd05aabf1cfc26a3da8cfc68c5/app/charts/shared/chart-dimensions.tsx#L73-L80

but it's only added in one condition at the end (if we don't have interactive elements). I think you should add this to the above function and consolidate the logic (next to `BRUSH_BOTTOM_SPACE`), instead of adding this in a few different places 👍